src/components/select/select.tsx

```diff
@@ -255,14 +255,14 @@ export class Select {
     ) {
         event.stopPropagation();
 
-        const selector = `#${this.portalId} limel-menu-surface limel-list`;
-        const list: HTMLElement = document.querySelector(selector);
+        const selector = `#${this.portalId} .mdc-menu-surface`;
+        const menuSurface: HTMLElement = document.querySelector(selector);
 
-        if (!list) {
+        if (!menuSurface) {
             return;
         }
 
-        const scrollPosition = list.scrollTop;
+        const scrollPosition = menuSurface.scrollTop;
```

Comment:
ah!! I should not use list there! but `menuSurface`

Reply:
Look at the whole fixup. That's "the same" variable, I just renamed it, because I changed so that it wasn't holding the list, but the menu-surface.

Reply:
Here's what the fixup looked like before I squashed it:

<img width="980" alt="image" src="https://github.com/user-attachments/assets/0dc83155-067f-4ed3-a705-bf33e62a6561">

Reply:
right, missed the top part.